tsconfig.json

```diff
@@ -2,29 +2,29 @@
   // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
   "include": ["src", "types"],
   "compilerOptions": {
-    "target": "ES5",
-    "module": "esnext",
-    "lib": ["dom", "esnext"],
-    "importHelpers": true,
+    "target": "ES2019",
```

Comment:
conservatively we could use here ES2018, according to TypeScript docu:
- ES2018 | Additional APIs available in ES2018 - **async iterables, promise.finally**, Intl.PluralRules, regexp.groups, 
- ES2019 | Additional APIs available in ES2019 - array.flat, array.flatMap, Object.fromEntries, string.trimStart, string.trimEnd, etc.

-> ES2019 does not bring much more for us against E2018... On the other hand its a new library and we have the year 2021...

Reply:
I've been using `.flatMap` and `.fromEntries` a lot more frequently lately, but I understand the reasoning. API polyfills are very easy for end-users to add. ES2019 support came with Safari 12.1 (March 2019).

Reply:
ok, lets try with ES2019